envisage/plugins/ipython_kernel/tests/test_internal_ipkernel.py

```diff
@@ -1,7 +1,12 @@
 import unittest
 
-from IPython.kernel.zmq.kernelapp import IPKernelApp
+try:
+    import IPython  # noqa
+except ImportError:
+    from nose.plugins.skip import SkipTest
+    raise SkipTest('IPython not available')
```

Comment:
IPython is not part of the hard dependencies of envisage, so the right thing to do is to skip tests that require an optional depedency. Furthermore, it looks that IPython does not support python 2.6 (even thought the package is installed, the tests are skipped).